common/nym-kcp/src/driver.rs

```diff
@@ -0,0 +1,60 @@
+use bytes::BytesMut;
+use log::{debug, trace};
+
+use crate::{error::KcpError, packet::KcpPacket, session::KcpSession};
+
+pub struct KcpDriver {
```

Comment:
Actually, what's the point of the `KcpDriver`? it doesn't appear to be used anywhere, even on the `drazen/gateway-probe-localnet-mode` branch?